contrib-nodes/node-red-contrib-dojot/README.md

```diff
@@ -0,0 +1,20 @@
+# middleware-IoT
```

Comment:
Uhm, I think that this updates (including the "name of the <organization>" a few lines up) could be included while improving the rest of the documentation.